ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -1994,3 +1995,15 @@ def get_logsize(host, logfile):
     """ get current logsize"""
     logsize = len(host.get_file_contents(logfile))
     return logsize
+
+
+def pkg_installer():
```

Comment:
The pkg installer should depend on the OS of the IPA node, not on the OS of the controller from where the test is launched.